src/Components/SmartComponents/CVEs/CVEsTableToolbar.js

```diff
@@ -54,20 +55,31 @@ const CVEsTableToolbarWithContext = ({ context, canEditStatusOrBusinessRisk, can
         ...canEditStatusOrBusinessRisk ? [
             {
                 label: intl.formatMessage(messages.editBusinessRisk),
-                onClick: () => methods.showBusinessRiskModal(selectedRowsRawData.map(({ id, attributes }) => ({
+                onClick: () => methods.showBusinessRiskModal(selectedCves.map(({
                     id,
-                    business_risk_id: attributes.business_risk_id,
-                    justification: attributes.business_risk_text
+                    business_risk_id,
+                    business_risk_text,
+                    attributes
+                }) => ({
+                    id,
+                    business_risk_id: attributes ? attributes.business_risk_id : business_risk_id,
+                    justification: attributes ? attributes.business_risk_text : business_risk_text
                 })), true),
                 props: { isDisabled: !selectedCvesCount }
             },
             {
                 label: intl.formatMessage(messages.editStatus),
-                onClick: () => methods.showStatusModal(selectedRowsRawData.map(({ id, attributes }) => ({
+                onClick: () => methods.showStatusModal(selectedCves.map(({
+                    id,
+                    exposed_systems_count,
+                    status_justification,
+                    status_id,
+                    attributes
+                }) => ({
                     id,
-                    exposed_systems_count: attributes.systems_affected,
-                    justification: attributes.status_text,
-                    status_id: attributes.status_id
+                    exposed_systems_count: attributes ? attributes.systems_affected : exposed_systems_count,
+                    justification: attributes ? attributes.status_text : status_justification,
+                    status_id: attributes ? attributes.status_id : status_id
```

Comment:
Why do selected CVEs contain info inside `attributes` and sometimes outside? I think it'd cleaner if we always had it outside.